zerver/lib/zulip_update_announcements.py

```diff
@@ -73,29 +75,33 @@ def internal_prep_group_direct_message_for_old_realm(
-        content = """\
-Zulip now supports [configuring]({organization_settings_url}) a stream where Zulip will \
-send [updates]({zulip_update_announcements_help_url}) about new Zulip features. \
-These notifications are currently turned off in your organization. If you configure \
+        content = remove_single_newlines(
+            """
+Zulip now supports [configuring]({organization_settings_url}) a stream where Zulip will
+send [updates]({zulip_update_announcements_help_url}) about new Zulip features.
+These notifications are currently turned off in your organization. If you configure
 a stream within one week, your organization will not miss any update messages.
 """.format(
-            zulip_update_announcements_help_url="/help/configure-automated-notices#zulip-update-announcements",
-            organization_settings_url="/#organization/organization-settings",
+                zulip_update_announcements_help_url="/help/configure-automated-notices#zulip-update-announcements",
+                organization_settings_url="/#organization/organization-settings",
+            )
         )
     else:
-        content = """\
-Starting tomorrow, users in your organization will receive [updates]({zulip_update_announcements_help_url}) \
+        content = remove_single_newlines(
+            """
+Starting tomorrow, users in your organization will receive [updates]({zulip_update_announcements_help_url})
 about new Zulip features in #**{zulip_update_announcements_stream}>{topic_name}**.
 
-If you like, you can [configure]({organization_settings_url}) a different stream for \
-these updates (and [move]({move_content_another_stream_help_url}) any updates sent before the \
+If you like, you can [configure]({organization_settings_url}) a different stream for
+these updates (and [move]({move_content_another_stream_help_url}) any updates sent before the
 configuration change), or [turn this feature off]({organization_settings_url}) altogether.
 """.format(
-            zulip_update_announcements_help_url="/help/configure-automated-notices#zulip-update-announcements",
-            zulip_update_announcements_stream=realm.zulip_update_announcements_stream.name,
-            topic_name=topic_name,
-            organization_settings_url="/#organization/organization-settings",
-            move_content_another_stream_help_url="/help/move-content-to-another-stream",
+                zulip_update_announcements_help_url="/help/configure-automated-notices#zulip-update-announcements",
+                zulip_update_announcements_stream=realm.zulip_update_announcements_stream.name,
+                topic_name=topic_name,
+                organization_settings_url="/#organization/organization-settings",
+                move_content_another_stream_help_url="/help/move-content-to-another-stream",
+            )
         )
     return internal_prep_huddle_message(realm, sender, content, recipient_users=administrators)
```

Comment:
Would it be better to just do `remove_single_newlines(content)` down here, and similarly in the `onboarding.py` code? It might really help avoid the excessive nesting that we're ending up with.

Reply:
Yeah, better.

I haven't updated `onboarding.py`, because other welcome messages are currently not in `"""` format. And we shouldn't change them to `"""` format here because we are going to remove/update a bunch of them as a part of #29421